contracts/examples/OracleCreator.sol

```diff
@@ -0,0 +1,183 @@
+pragma solidity =0.6.6;
+
+import './../libraries/TransferHelper.sol';
+import './../libraries/DXswapOracleLibrary.sol';
+import './../libraries/DXswapLibrary.sol';
+import './../libraries/SafeMath.sol';
+
+contract OracleCreator {    
+    using FixedPoint for *;
+    using SafeMath for uint256;
+
+    event OracleCreated(
+        uint256 _oracleId,
+        address _pair,
+        uint256 _windowSize,
+        uint8 _granularity,
+        uint256 _periodSize,
+        uint256 _executionBountyWei
+    );
+
+    event OracleExecuted(
+        uint256 _oracleId
+    );
+
+    struct Oracle{
+      uint256 windowSize;
```

Comment:
WindowSize could be rename to WindowTime, same foll all variables in Oracle, I think it would be a better name since the "size" of the window is determined by its time, and thats what we are using, "windows of time" or "periods of time" maybe time period would be even better.